gems/smithy-client/lib/smithy-client/endpoint_rules.rb

```diff
@@ -105,7 +105,9 @@ def self.attr(value, path)
         if parts.size == 1
           val
         else
-          attr_reader(val, parts.slice(1..-1).join('.'))
+          # rubocop:disable Style/Attr
```

Comment:
You can single line comment this and not need to disable/enable but only disable.